### PR TITLE
Add Maven version check

### PR DESCRIPTION
**Pull request by Virtlink**
Add a Maven version check to `./b`.

Depends on a 0.2 release of `mavenpy`, [pull request #1](https://github.com/Gohla/mavenpy/pull/1)

### CODE REVIEW

File: releng/metaborg/releng/cmd.py [hunk 1]
```diff
@@ -57,6 +59,9 @@ def get_list(self, key, defaults=None):
 class MetaborgReleng(cli.Application):
   PROGNAME = 'b'
 
+  minimumMvnVersion = version.parse("3.5.4")
+  forbiddenMvnVersions = [version.parse("3.6.1")]
```
Comment (mpsijm) [inline releng/metaborg/releng/cmd.py]: According to https://github.com/archimatetool/archi/issues/513#issuecomment-558558002, Maven version 3.6.2 could also give problems. I did not test it with this version though

Comment (mpsijm) [reply releng/metaborg/releng/cmd.py]: Update: tested with 3.6.2, this is also broken (and indeed gives an error related to polyglot, as indicated in the issue I linked above)

Comment (Virtlink) [reply releng/metaborg/releng/cmd.py]: Thanks! I added 3.6.2 to the list of broken Maven versions.

**Comment by Virtlink** [comment]
Updated the version.

**Comment by Gohla** [comment]
Thanks!

**Pull request merged**